components/shared/Support.vue

```diff
@@ -16,13 +18,15 @@ import { round } from '@/utils/support'
 @Component({})
 export default class Support extends Vue {
   @Prop() public value!: boolean
+  @Prop({ type: Boolean, default: true }) public showPrice?: boolean
   @Prop({ default: 0 }) public price!: number
-  @Prop({ default: 'I\'m helping to cover costs' }) public activeMessage!: string
-  @Prop({ default: 'I don\'t want to support' }) public passiveMessage!: string
+  @Prop({ default: 'I am helping to cover costs' })
```

Comment:
also may move this to translations

Reply:
you cannot use `$t` in defaults as far as i'm aware, since context isn't available to Props during initialisation.
however it gets translated in the template above anyways.